lib/bulk_processor/back_end/gcp_pods.rb

```diff
@@ -0,0 +1,27 @@
+require 'gcp_job_manager'
+
+class BulkProcessor
+  module BackEnd
+    class GcpPods
+      def initialize(processor_class:, payload:, key:)
+        @processor_class = processor_class.name
+        @payload = PayloadSerializer.serialize(payload)
+        @key = key
+      end
+
+      def start
+        args = [processor_class, payload, key]
+        GcpJobManager.create_and_deploy_job('start-bulk-processor', args)
```

Comment:
were you saying we would need to copy/paste these jobs in every service's config? I think there might be a way to make them added as part of this gem.

Reply:
👍